### PR TITLE
Feat: Use `shiny.semantic` to improve app UI

### PR DESCRIPTION
**Pull request by vibalre**
* App should now use `shiny.semantic` to make the app look cleaner.
* There should be input for taxonomy ID and species name and selection for cell type.
* Coloring the cell should now be accompanied by a table that renders the name of the subcellular location and the color it has.
* Button below the color picker should clear the cell of colors.

### CODE REVIEW

File: inst/shinyApp/drawCellShiny/server.R [hunk 1]
```diff
@@ -1,59 +1,130 @@
...
     sc_id(substr(input$cell_click, 3, 6))
-    
-    list_named_colours <- c(subcelular_colours(), input$colourInput)
-    names(list_named_colours)[length(list_named_colours)] <- input$cell_click
-    subcelular_colours(list_named_colours)
+
+    list_named_colours <- subcellular_colours()
+    list_named_colours[[input$cell_click]] <- input$colourInput
+    subcellular_colours(list_named_colours)
   })
```
Comment (svalvaro) [inline inst/shinyApp/drawCellShiny/server.R:42]: Nice!, I assume this would prevent having repeated rows in the table.

File: DESCRIPTION [hunk 2]
```diff
@@ -1,7 +1,7 @@
 Package: drawCell
 Type: Package
 Title: Generation of cell pictures
-Version: 0.99.0
+Version: 1.0.0.9000
 Authors@R: c(person(given = "Alvaro", family = "Sanchez-Villalba",
```
Comment (svalvaro) [inline DESCRIPTION:5]: Feel free to add yourself as an author!

File: NEWS.md [hunk 3]
```diff
@@ -1,4 +1,5 @@
-# drawCell 0.99.0
+# drawCell (development version)
 
 
 * Submission to Bioconductor 3.14
+* Improve shiny app with `shiny.semantic`
```
Comment (svalvaro) [inline NEWS.md:5]: We need to add here that now the pictures are vector images (higher quality).
Besides we also allow multi colouring, a feature that I did not implement before.

**Review by svalvaro — APPROVED** [review]
This is very clean PR, really good job. It works really well, the code is very clean.
Not much to improve in my opinion.

I will implement the few suggestions msyelf.

**Comment by svalvaro** [comment]
@vibalre Awesome job on this! 💯 It works really great on my side, I don't really have any issue with the size of the cell in my browser.

I have some suggestions like making the text/labels bigger, and centered. Though I can do that myself in a new PR.

**Comment by svalvaro** [comment]
If you're done with that branch, you can delete it @vibalre

**Comment by vibalre** [comment]
Thanks for the merge. If I have any updates, I will create a new branch and pull request.

**Pull request merged**